<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>题目</string>
   </property>
  </widget>
  <widget class="QLabel" name="content">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>421</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>请先连接服务器</string>
   </property>
  </widget>
  <widget class="QPushButton" name="commit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>提交答案</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="buttonA">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>411</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>选项A</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="buttonB">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>选项B</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="buttonC">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>选项C</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="buttonD">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>选项D</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>30</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器ip地址:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ipAddr">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="port">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>81</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器端口:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectServer">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>130</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <widget class="QPushButton" name="disconnectServer">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>130</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>断开</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>请作答（单选题）：</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
